packages/account/README.md

```diff
@@ -38,13 +38,16 @@ pnpm add fuels
 npm add fuels
 ```
 
-# Test Utilities
+## Test Utilities
```

Comment:
Just curious but why the change to h2?

Reply:
That's reasonable. Good change. I like that you referenced the HTML standard as well.